### PR TITLE
feat/(CORSO-76): update landing page UI design 

### PR DESCRIPTION
**Pull request by KarinaSofia**
JIRA: https://champlainsaintlambert.atlassian.net/browse/CORSO-76
## Context:
Updated the ui of the landing page and implemented more localisation
## Changes
updated the following components: 
-gallery
-view pinned faq
-view pinned review
-view services
-footer
-banner
## Before and After UI (Required for UI-impacting PRs)
image
image
image
image
image
image

### CODE REVIEW

File: corso_website_fe/package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "react-dom": "^18.2.0",
     "react-google-autocomplete": "^2.7.3",
```
Comment (DylanBrass) [inline corso_website_fe/package.json:36]: Don't forget to uninstall the previous carousel dependency

File: corso_website_fe/src/Pages/MainPage/Main Page Components/Gallery/OurPreviousProjects.tsx [hunk 2]
```diff
@@ -17,47 +16,85 @@ interface ImageData {
...
             try {
-                const response = await axios.get(process.env.REACT_APP_BE_HOST + 'api/v1/corso/galleries/carousel');
-                
-                setImages(response.data);
+                const response = await axios.get<ImageData[]>(`${process.env.REACT_APP_BE_HOST}api/v1/corso/galleries/carousel`);
+                const sortedImages = response.data.sort((a, b) => a.image_position - b.image_position);
+                setImages(sortedImages);
             } catch (error) {
-                
+                // Handle error
```
Comment (DylanBrass) [inline corso_website_fe/src/Pages/MainPage/Main Page Components/Gallery/OurPreviousProjects.tsx]: Don't forget to put the auth error handler here.

File: corso_website_fe/src/Pages/MainPage/Main Page Components/Gallery/OurPreviousProjects.tsx [hunk 3]
```diff
@@ -17,47 +16,85 @@ interface ImageData {
...
             }
         };
-        fetchImages()
+        fetchImages();
```
Comment (DylanBrass) [inline corso_website_fe/src/Pages/MainPage/Main Page Components/Gallery/OurPreviousProjects.tsx:31]: William would be disappointed

**Review by DylanBrass — DISMISSED** [review]
Looks good, one thing is idk how responsive it is, did you test it and if so send pictures pls.

**Review by DylanBrass — APPROVED** [review]

File: corso_website_fe/src/Pages/MainPage/Main Page Components/Reviews/ReviewPinned.tsx [hunk 4]
```diff
@@ -1,78 +1,93 @@
...
-    useEffect(() => {
-        getPinnedReviews()
-        // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, []);
+    const responsive = {
+        superLargeDesktop: { breakpoint: { max: 4000, min: 0 }, items: 1 },
+        desktop: { breakpoint: { max: 4000, min: 0 }, items: 1 },
+        tablet: { breakpoint: { max: 4000, min: 0 }, items: 1 },
+        mobile: { breakpoint: { max: 4000, min: 0 }, items: 1 }
+    };
```
Comment (Mila5847) [inline corso_website_fe/src/Pages/MainPage/Main Page Components/Reviews/ReviewPinned.tsx:38]: I see you made it responsive here but as mentioned previously some screenshots would help see how it looks like

**Review by Mila5847 — APPROVED** [review]
Looks good as design! My only concerns were already expressed by our teammate

**Pull request merged**